src/routes/permissions.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import express from 'express';
+import db from '../models/db';
+
+let router = express.Router();
+
+// Return the list of allowed permissions.
+router.get('/', (req, res) => {
+  db().then(models => {
+    models.Permissions.findAll().then(permissions => {
+      res.status(200).json({
+        permissions: permissions.map(permission => permission.name)
+      });
+    });
+  });
+});
```

Comment:
nit: you can cut down the callback pyramid like this:
```javascript
router.get('/', (req, res) => {
  db()
    .then(models => models.Permissions.findAll())
    .then(permissions =>
      res.status(200).json({
        permissions: permissions.map(permission => permission.name)
      })
    );
});
```

(indentation, up to you :) )

Because we don't need `models` in the second callback, we can let the promise flow back to the upper level.